src/rateEngine/LoadProfileScaler.ts

```diff
@@ -48,6 +48,16 @@ class LoadProfileScaler {
     }
   }
 
+  toMonthlyKwh(monthlyKwh: Array<number>): LoadProfile {
+    const scalersByMonth = this.loadProfile.sumByMonth().map((kwh, idx) => {
+      return monthlyKwh[idx] / kwh;
+    });
+    const scaledLoad = this.loadProfile.expanded().map(loadHour => {
+      return loadHour.load * scalersByMonth[loadHour.month];
+    });
+    return new LoadProfile(scaledLoad, {year: this.loadProfile.year});
```

Comment:
It almost certainly doesn't matter at this scale, but it's worth pointing out here that if you pass a new LoadProfile object an expanded load profile, you save it some work later on.

Reply:
great advice. the scale at which this matters probably isn't that great.. i changed it